give/src/components/Title/Title.js

```diff
@@ -0,0 +1,70 @@
+// Based on v3
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/title.html
+// https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/title.html
+
+import "./Title.css";
+
+const LinkWrapper = ({ href, children }) => {
+  return href ? (
+    <a className="cmp-title__link" href={href}>
+      {children}
+    </a>
+  ) : (
+    children
+  );
+};
+
+export const Title = ({
+  className = "", // string
+  component = "", // string = "h1" | "h2" | "h3" | "h4" | "h5" | "h6"
```

Comment:
Component seems like a weird name for this. When I goggle something like scaleCategory or category might be a way to define it.

Reply:
The `variant` prop sets the style, but the `component` prop sets the element. So `<Typography component="h1" variant="h2">...</Typography>` would render an `<h1>...</h1>` that's styled like the default `<h2 />`. I believe I'm doing something like this on my personal preferences screen for MPDx.

However, for this particular project, I just renamed my prop `element` to be explicit about the type of content it's expecting.

Reply:
https://github.com/CruGlobal/cru-content-designs/pull/8/commits/88e6912290a450091d57cf2aab195aaa31953570